Dev ui

xcixor opened the pull request:
## Purpose
Created the front-end framework for the application and is almost ready to merge. Review and tell me what you think.

## Tasks covered

- Created signup page
- Created Login page
- Created user dashboard
- Created Business Catalog
- Added means to review a business
- Added means to view reviews on a business
- Added means to perform CRUD on a business

xcixor committed:
Chore: Move html files and css files to relevant folders

xcixor committed:
[Feature: #155499155] Style the landing page

The landing page needs to be attractive and its content precise to attract customers. To achieve more aesthetics a background image was added, more colors used on various components and icons added to the form

xcixor committed:
Fix: Place email field before password field

xcixor committed:
[Feature: #155502027] Create files for login page

xcixor committed:
Refactor: Change code to accomodate changes new login feature

xcixor committed:
[Feature: #155502027] Add login form

This form will be used to collect login information so that users can be granted access to their accounts

xcixor committed:
Ignore file

Its contents do not contribute directly to the development of the app

xcixor committed:
[Feature #155494152] Create files for dashboard

The dashboard will contains the component for adding businesses and its where a business will appear after creation. Furthermore it will have a component for searching and filtering a business according to prefered criteria

xcixor committed:
[Feature: #155494152] Add js file

It will contain initializations for materialize frameworl components

xcixor committed:
[Feature: #155494152] Initialize side nav

xcixor committed:
[Feature: #155494152] Create side nav and top nav bar

The side nav will contain links to user's profile, created business and sign out. The top nav will contain a search field for businesses

xcixor committed:
[Feature: #155494152] Add create business button

By clicking this button the user will be able to create a business

xcixor committed:
[Feature: #155494152] Add business creation form

The form will be used to collect data for a business

xcixor committed:
[Feature: #155494152] Add a mock business

The mock business show icons for performing crud operations that would be used by a real business

xcixor committed:
[Feature: #155498815] Added component for viewing reviews

xcixor committed:
Fix: Add spacing around FAB to make attractive

xcixor committed:
[Feature: #155494626] Create files for editing the business profile

xcixor committed:
[Feature: #155494626] Add form for editing the business profile

xcixor committed:
Fix: Add more fields for collecting user data

xcixor committed:
[Feature: #155494626] Add link editing the business profile

xcixor committed:
Ignore unnecessary file

xcixor committed:
Merge branch 'dev-dashboard' into dev-ui

xcixor committed:
[Feature: #155499070] Add files for creating the catalog

xcixor committed:
[Feature: #155497671] Add component for searching for a business in the catalog

xcixor committed:
[Feature: #155499070] Add filter for category

The user needs to filter the businesses by category. This was achieved by using tabbed panes

xcixor committed:
[Feature: #155499050] Add filter for location

This will enable the user to filter the businesses by location

xcixor committed:
[Fix] change color of navbar

xcixor committed:
Fix: Edit the look and feel of the catalog

xcixor committed:
Fix: Remove unnecesary links

xcixor committed:
[Feature] Add link to business catalog

xcixor committed:
Add details to README.md

The details will assist potential users of the app how to utilize it

xcixor committed:
Fix: edit link to github pages

xcixor committed:
Fix: Edit github pages link

migot01 commented:
Looks good, nice touch on the glyphicons

miritih commented:
Looks Good to me. I like how you used materialize.
One thing I have noticed, there is no way to navigate to other pages.  Everything else  looks good 👍

xcixor committed:
Chore: update on installation process

xcixor committed:
Fix: change links to full path to file

Changes were requested on PR to provide link to all pages. Links were thus added where they did nit exist or existing ones changed to full path to files to provide access to resources

xcixor committed:
Fix: Increase link font size and change color

xcixor committed:
Feature: Create navbar

A top navbar will ease access to other pages.
